pkg/virt-controller/network/pod_annotations.go

```diff
@@ -31,19 +31,35 @@ import (
 )
 
 func GeneratePodAnnotations(networks []virtv1.Network, interfaces []virtv1.Interface, multusStatusAnnotation string, bindingPlugins map[string]virtv1.InterfaceBindingPlugin) map[string]string {
-	newAnnotations := map[string]string{}
+	var err error
+
+	var sriovIfacePciMap map[string]*networkv1.DeviceInfo
 	if vmispec.SRIOVInterfaceExist(interfaces) {
-		networkPCIMapAnnotationValue := deviceinfo.CreateNetworkPCIAnnotationValue(
-			networks, interfaces, multusStatusAnnotation,
-		)
-		newAnnotations[deviceinfo.NetworkPCIMapAnnot] = networkPCIMapAnnotationValue
+		sriovIfacePciMap, err = deviceinfo.MapNetworkNameToDeviceInfo(networks, multusStatusAnnotation, interfaces)
```

Comment:
I do not see the SR-IOV interfaces being filtered.
I think you need an equivalent `MapBindingPluginNetworkNameToDeviceInfo` helper, e.g. `MapSRIOVNetworkNameToDeviceInfo`.

Reply:
I followed the current state, note that SR-IOV interfaces were not filtered as well.
`sriovIfacePciMap` is bad name and doesnt reflect it.

If SR-IOV interfaces are filtered here, the end result is network-info annotation have only SR-IOV and binding-plugin with device-info interfaces.
Other interfaces that use different bindings will not be included.

Reply:
> I followed the current state, note that SR-IOV interfaces were not filtered as well. `sriovIfacePciMap` is bad name and doesnt reflect it.

Per what I see, they have been filtered [1] and you removed it with this change.

[1] https://github.com/kubevirt/kubevirt/blob/a72353209753b177a2d71abc7204c6254befc89a/pkg/network/deviceinfo/sriov.go#L60

> If SR-IOV interfaces are filtered here, the end result is network-info annotation have only SR-IOV and binding-plugin with device-info interfaces. Other interfaces that use different bindings will not be included.

Who said they need to be included?
The PR is focused on SR-IOV, please keep it focused on it or reason for something else.
If you want to generalize it, it will be easier to open a different PR and convince us it makes sense.